Reactions/ReactorBase.H

```diff
@@ -78,7 +80,15 @@ public:
     long int* FCunt,
     amrex::Real dt) = 0;
 
-  virtual void SetTypValsODE(const std::vector<amrex::Real>& ExtTypVals) = 0;
+  void SetTypValsODE(const std::vector<amrex::Real>& ExtTypVals);
```

Comment:
I did this although now the application has to make this change as well. @esclapez - you may have to change LM

Reply:
hmmm didn't think about that. Maybe that's a bad idea?

Reply:
We don't really have naming convention, but I like no capital on the first letter and capital on each of the word. But this is neither ! We'll need to update LM if this changes, but it's not really a problem.

Reply:
We have no style guide! What a mess haha. I've been using lowercase and `_` for the reactor classes member and functions so that's the only reason I suggested the change...